docs/content/en/api/query-builder-methods.md

```diff
@@ -15,6 +15,18 @@ Eager load relationships.
 await Model.include('user', 'category')
 ```
 
+## `with`
+- Arguments: `(...args)`
+- Returns: `self`
+
+Eager load relationships.
+
+```js
+await Model.with('user', 'category')
+```
+
+<alert type="info">This method is an alias of [include](/api/query-builder-methods#include)</alert>
+
```

Comment:
I think we should “avoid” duplicated docs. 

Would be enough to alert on “get()” docs that there is a alias “all”. 

Same for with/include

Reply:
Ok!

Reply:
Done in b932e9b